piv/key.go

```diff
@@ -102,6 +103,9 @@ type Attestation struct {
 	PINPolicy PINPolicy
 	// TouchPolicy set on the slot.
 	TouchPolicy TouchPolicy
+
+	// Slot is the name of the slot the attested key resides in
```

Comment:
nit: add a period to the end of the comment.

What are the set of names this could be? "sign" or a hex number? Can you add an example to the comment?